app/services/withdraw_manual_service.rb

```diff
@@ -0,0 +1,38 @@
+class WithdrawManualService
+  def initialize(manual_repository:, listeners: [], manual_id:)
+    @manual_repository = manual_repository
+    @listeners = listeners
+    @manual_id = manual_id
+  end
+
+  def call
+    withdraw
+    persist
+    notify_listeners
```

Comment:
If you try to withdraw a draft manual, then its state won't be updated but the listeners will still be called, so a 'gone' content item will be put into the content store and so a 'gone' page will be served instead of a 404.